bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
-CGINS-METCTD-12756,A01268,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12756,,20141214,5816,,
-CGINS-METCTD-12757,A01269,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12757,,20141214,5816,,
-CGINS-METCTD-12758,A01270,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12758,,20141214,5816,,
+CGINS-METCTD-08028,A00207,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37SM61212-8028,,20101122,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-10137,A00190,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10137,,20130131,5816,Missing from inventory,Woods Hole Oceanographic Institution
+CGINS-METCTD-10138,A00197,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10138,,20121226,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-10139,A00204,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10139,,20130131,5816,Missing from inventory,Woods Hole Oceanographic Institution
+CGINS-METCTD-10903,A00500,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10903,,20131101,5816,,Oregon State University
+CGINS-METCTD-10904,A00476,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10904,,20131021,5816,,Oregon State University
+CGINS-METCTD-10905,A00499,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10905,,20131101,5816,,Oregon State University
+CGINS-METCTD-10906,A00498,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10906,,20131101,5816.32,,Oregon State University
+CGINS-METCTD-11000,A00120,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11000,,20121226,5816,,Oregon State University
+CGINS-METCTD-11001,A00121,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11001,,20150604,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-11002,A00477,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11002,,20131021,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-11003,A00440,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11003,,20130930,5816,,Woods Hole Oceanographic Institution
```

Comment:
For 11003, add Comment "Lost at sea on GI01SUMO-00004".

Reply:
done